code/datums/religions.dm

```diff
@@ -1179,7 +1222,7 @@
 	bible_name =  "Honkmothers Coloring Book"
 	male_adept = "Co-Clown"
 	female_adept = "Co-Clown"
-	keys = list("honk", "clown", "honkmother")
+	keys = list("honk", "clown", "shoe snatchin willy", "slip", "banana", "bananas", "slipping", "shoe snatching")
```

Comment:
Why did you remove "honkmother" from the list? Also, add aunt scootaloo.

Reply:
because its already a deity name